arrow-core/src/main/kotlin/arrow/core/extensions/setk/foldable/SetKFoldable.kt

```diff
@@ -96,8 +96,9 @@ fun <A> Kind<ForSetK, A>.fold(arg1: Monoid<A>): A = arrow.core.SetK.foldable().r
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "reduceLeftToOption(arg1, arg2)",
-    "arrow.core.reduceLeftToOption"
+    "Option.fromNullable(reduceNullable(arg1, arg2))",
+    "arrow.core.reduceNullable",
```

Comment:
```suggestion
    "Option.fromNullable(reduceOrNull(arg1, arg2))",
    "arrow.core.reduceOrNull",
```

Also needs to be fixed for `ListK` and `MapK` 🙏